jpo-ode-common/src/test/java/us/dot/its/jpo/ode/inet/InetPointTest.java

```diff
@@ -53,13 +53,13 @@ public void testStringConstructorCreatesAddress() {
             }
          };
 
-         new InetPoint("hostString123", 5, true);
+         new InetPoint("bah.com", 5, true);
```

Comment:
Why is "bah.com" hardcoded here?

Reply:
This is unit testing against a particular condition with a host name for unit testing InetPoint class.